packages/devtools_extensions/example/README.md

```diff
@@ -1,4 +1,16 @@
 # DevTools extension examples
+
+## Run this example
+
+1. Start app_that_uses_foo on any platform
+2. Open DevTools with one of the following ways:
+  - button 'DevTools' on Flutter panel in VS Code
+  - menu {} > Dart DevTools at the bottom of VS Code
+  - with command `devtools_tool serve .` in `packages/devtools_app` of DevTools repo
```

Comment:
instead of duplicating instructions, let's instead link to the `app_that_uses_foo` section below.